pyrgg/__main__.py

```diff
@@ -66,7 +68,8 @@ def gen_graph(input_dict, file_name):
         sign=sign,
         direct=direct,
         self_loop=self_loop,
-        multigraph=multigraph)
+        multigraph=multigraph,
+        probability=probability)
```

Comment:
Each model should have specific inputs. For example, the `Erdős-Rényi-Gilbert` model only requires `number of vertices` and `probability` as inputs.

If we need to define separate functions, I suggest moving each model's functions to a separate file (`pyrgg_engine.py`, `erdos_reny_gilbert_engine.py`, etc)

Reply:
Don't you think it's better to handle this one separately in another PR?